activiti-cloud-service-common/activiti-cloud-services-test/src/main/java/org/activiti/cloud/starters/test/builder/ProcessInstanceEventContainedBuilder.java

```diff
@@ -62,6 +64,16 @@ private ProcessInstanceImpl buildProcessInstance(String name) {
         return completedProcess;
     }
 
+    private CloudProcessInstanceImpl buildSuspendedProcessInstance(String name) {
+        CloudProcessInstanceImpl suspendedProcess = new CloudProcessInstanceImpl();
+        suspendedProcess.setId(UUID.randomUUID().toString());
+        suspendedProcess.setName(name);
+        suspendedProcess.setProcessDefinitionKey("my-proc");
+        suspendedProcess.setProcessDefinitionId(UUID.randomUUID().toString());
+        suspendedProcess.setProcessDefinitionName("my-proc-definition-name");
```

Comment:
It's probably better to set the status to `SUSPENDED` as well.

Reply:
Thanks, added.